doc/docs/modules/ROOT/pages/overview.adoc

```diff
@@ -20,7 +20,7 @@ and Scala.
 == Compatibility
 
 === Neo4j compatibility
-This connector works with Neo4j 3.5 and the entire 4.x series of Neo4j, whether run as a single instance,
+This connector works with Neo4j 3.5 and the entire 4.x and 5.x series of Neo4j, whether run as a single instance,
```

Comment:
we should maybe remove the mentions of `3.5`?

Reply:
fixed in 63701c36f8dc836271df379f830eee78c157a2db